xbanish.c

```diff
@@ -444,10 +442,10 @@ snoop_legacy(Window win)
 }
 
 void
-usage(void)
+usage(char *progname)
```

Comment:
Lift to top of file?

Reply:
I don't quite understand why we'd need to lift it to the top of the file. For stylistic reasons?

Reply:
Yes. And this eliminates the unnecessary definition of the ```usage``` function.